src/core/Akka.Cluster.Tests/ActorRefProvidersConfigSpec.cs

```diff
@@ -40,11 +40,13 @@ public void ActorProviderConfig_should_resolve_cluster_alias()
 
         private void ConfigureAndVerify(string alias, Type actorProviderType)
         {
-            var config = ConfigurationFactory.ParseString(@"akka.actor.provider = " + alias);
+            var config = ConfigurationFactory.ParseString(@"akka.actor.provider = " + alias)
+                .WithFallback(ConfigurationFactory.ParseString("akka.remote.dot-netty.tcp.port = 0")); // use a random port to avoid issues with async and parallelization
```

Comment:
Resolved via port 0 binding (won't have any side-effects on using a local `ActorSystem` since the configuration never gets read.)